tests/unittest_decorators.py

```diff
@@ -97,3 +99,18 @@ def test_deprecated_default_argument_values_ok(recwarn: WarningsRecorder) -> Non
         instance = SomeClass(name="some_name")
         instance.func(name="", var=42)
         assert len(recwarn) == 0
+
+
+@pytest.mark.skipif(sys.version_info < (3, 8), reason="Requires Python 3.8 or higher")
```

Comment:
```suggestion
@pytest.mark.skipif(not PY38_PLUS, reason="Requires Python 3.8 or higher")
```
For tests it isn't necessary to use `sys.version_info` (for typing).